examples/arrays/arrays.md

```diff
@@ -4,6 +4,21 @@ An array can be used to hold a set of values of the same type. The array type ca
 
 The length of the array can be inferred from the context by defining the array as `T[*]`. The length of the array should be known in compile time.
 
+# Quick Comparison of Array Types in Ballerina
+
+| **Array Type**             | **Declaration Syntax** | **Example Syntax**                  | **Length Known at Compile Time** | **Resizable** |
+|----------------------|------------------------|-------------------------------------|----------------------------------|---------------|
+| Fixed-length   | `T[n] <arrayName>;`               | `int[3] nums = [1, 2, 3];`          | Yes ✅                           | No ❌         |
+| Variable-length      | `T[] <arrayName>;`                | `int[] even = [2, 4];`           | No ❌                            | Yes ✅        |
+| Inferred-length      | `T[*] <arrayName>;`            | `string[*] colors = ["red"];`       | Yes ✅                           | No ❌         |
```

Comment:
Since inferred-length arrays are also fixed-length arrays, I'm not sure this classification works. And if it's just fixed vs variable length, I think the descriptions work without the table too.